src/main/java/edu/tamu/sage/ui/controller/ViewController.java

```diff
@@ -5,11 +5,20 @@
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.servlet.ModelAndView;
+import edu.tamu.sage.service.SimpleProcessorService;
 
 // TODO: this most likely can move into the framework
 @Controller
 public class ViewController {
 
+    // private final SimpleProcessorService simpleprocessorService;
+    private static SimpleProcessorService simpleProcessorService;
+
+
+    public ViewController(SimpleProcessorService simpleProcessorService) {
+        ViewController.simpleProcessorService = simpleProcessorService;
+    }
+
```

Comment:
Can this not be `@Autowired`?

Reply:
Yes, It could. I went ahead and modified this controller and added the property and removed using the service. https://github.com/TAMULib/SAGE/commit/c3f00acde85f30148fa7c87e548bd03a3d61c113